casepro/pods/base.py

```diff
@@ -0,0 +1,31 @@
+from confmodel import fields, Config as ConfmodelConfig
+from django.apps import AppConfig
+
+
+class PodPlugin(AppConfig):
+    pass
+
+
+class PodConfig(ConfmodelConfig):
+    '''
+    This is the config that all pods should use as the base for their own
+    config.
+    '''
+    index = fields.ConfigInt(
+        "A unique identifier for the specific instance of this pod."
+        "Automatically determined and set in the pod registry.",
+        required=True)
+
+
+class Pod(object):
+    '''
+    The base class for all pod plugins.
+    '''
+    config_cls = PodConfig
+    url_patterns = []
```

Comment:
Addressed in https://github.com/praekelt/casepro/pull/31/commits/10f4b67e9d5d618f8c256c7b850bdc54a489ddab